src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -117,29 +113,29 @@ export class NativeStakingMapper {
         symbol: chain.nativeCurrency.symbol,
         trusted: true,
       }),
-      validators: args.transaction?.executionDate ? publicKeys : null,
+      validators: publicKeys,
     });
   }
 
   /**
    * Gets the validator public keys from logs of `deposit` transaction.
    *
-   * @param args.transaction - the transaction object for the deposit
+   * @param args.txHash - the transaction hash of the deposit transaction
    * @param args.chainId - the chain ID of the native staking deployment
    *
-   * @returns {Array<`0x${string}`>} the validator public keys
+   * @returns {Array<`0x${string}`> | null} the public keys of the validators
    */
   private async getDepositPublicKeys(args: {
-    transaction: MultisigTransaction | ModuleTransaction | null;
+    txHash: `0x${string}` | null;
     chainId: string;
-  }): Promise<Array<`0x${string}`>> {
-    if (!args.transaction?.transactionHash) {
-      return [];
+  }): Promise<Array<`0x${string}`> | null> {
+    if (!args.txHash) {
+      return null;
```

Comment:
I will consult with clients before merge this. If there is no hash, there is technically no keys so this is (currently) incorrect.